docs/components/connectors/out-of-the-box-connectors/available-connectors-overview.md

```diff
@@ -50,6 +50,7 @@ import TabItem from "@theme/TabItem";
 - [Salesforce Connector](/components/connectors/out-of-the-box-connectors/salesforce.md) - Manage your Salesforce instance from your BPMN process.
 - [SendGrid Connector](/components/connectors/out-of-the-box-connectors/sendgrid.md) - Quickly send emails from your BPMN processes.
 - [Slack outbound Connector](/components/connectors/out-of-the-box-connectors/slack.md) - Send messages to channels or users in your [Slack](https://slack.com) workspace from your BPMN process.
+- [SQL Connector](/components/connectors/out-of-the-box-connectors/sql.md) - Connect your BPMN process with SQL databases (Microsoft SQL Server, Postgre, MySQL).
```

Comment:
Instead of Postgre, MySQL, I think we mean "PostgreSQL"?

Reply:
updated